docs/steps.txt

```diff
@@ -4,20 +4,55 @@ ________________
 
 Task 1:
 
+First of all I created a pom file dependency file that has all of the dependency i need (web app, mongo database , health checker)
```

Comment:
please provide this document with proper steps to how-to run everything.

example:

> to start up the docker-compose run the following command:
> $ docker-compose up .